simulation_parameters/Constants.cs

```diff
@@ -516,6 +516,14 @@ public static class Constants
 
 #pragma warning disable CA1823 // unused fields
 
+    // Defines the thrust vectors given by the flagella, based on the flagella's orientation
+    public static readonly Vector3 VectorDown = GetVector(0);
+    public static readonly Vector3 VectorDownLeft = GetVector(1);
+    public static readonly Vector3 VectorUpLeft = GetVector(2);
+    public static readonly Vector3 VectorUp = GetVector(3);
+    public static readonly Vector3 VectorUpRight = GetVector(4);
+    public static readonly Vector3 VectorDownRight = GetVector(5);
```

Comment:
This isn't exactly the same as the version number property, these need to use the same two variable approach as it does, because I read that's the only way to get the lazy initialization work like that.

Reply:
I think I'm completely lost on this one... what is the "version number property", the "two variable approach", and the "lazy initialization" that you are referring to?